### DISCUSSION TITLE
fawe not working

### DISCUSSION THREAD

**Discussion by Shzv** [diamond-rip/Eden#85] [Q&A] [answered]
I have tried various solutions to get fawe working but I can't because old versions like 1.8.8 are not supported. Can someone help me? Thank you

**Comment by HierarchyMan** [comment]
Use older version of fawe
https://ci.athion.net/job/FastAsyncWorldEdit-Legacy/lastSuccessfulBuild/artifact/target/FastAsyncWorldEdit-bukkit-21.03.26-5ff3a9b-1286-22.3.9.jar
This version works for me

**Reply by Shzv** [reply]
i tried but i don't know why but it makes me crash the server

**Reply by HierarchyMan** [reply]
Delete old woldedit jar in plugins folder
Delete all extra jars you see

**Reply by Shzv** [reply]
keeps crashing my server. i started the server with fawe protocolib and eden.

**Comment by Shzv** [comment, marked as answer]
however I don't know why Fare does this. I guess I will continue to use eden without fawe thanks for the support

**Reply by GoodestEnglish** [reply]
Send me the error you have, I may try to help you to solve this.

**Reply by Shzv** [reply]
sorry i just read this is the link https://paste.me/paste/4b358abb-43ed-493c-67c7-b9fd0f29b486#dc7d5a15a794a6441fd81b34f413a3d5beb80836e77827f36ecedca606f16ca8

**Reply by GoodestEnglish** [reply]
What java version are you using?

**Reply by Shzv** [reply]
i use java 8 but i also tried with java 17 and i use papermc 1.8.8

**Reply by GoodestEnglish** [reply]
The error looks like you are not using java 8. Make sure you are using java 8 and if yes, then you might need to contact fawe developer

**Reply by Shzv** [reply]
ok i'm stupid i used the wrong java 8 because i have to install javaSE 8 now it goes
